lib/devicetrust/authz/authz.go

```diff
@@ -45,8 +47,8 @@ func IsTLSDeviceVerified(ext *tlsca.DeviceExtensions) bool {
 
 // VerifyTLSUser verifies if the TLS identity has the required extensions to
 // fulfill the device trust configuration.
-func VerifyTLSUser(dt *types.DeviceTrust, identity tlsca.Identity) error {
-	return verifyDeviceExtensions(dt, identity.Username, IsTLSDeviceVerified(&identity.DeviceExtensions))
+func VerifyTLSUser(ctx context.Context, dt *types.DeviceTrust, identity tlsca.Identity) error {
```

Comment:
I'm not sure passing the context here is the right call - it isn't respected or used in any way, apart from logging. I would log with context.Background() instead.

Reply:
I'm not convinced that maybe augmenting a debug log is worth the change in signature. I would expect a method that takes a context to respect it, or at least use something that does. I don't think just logging qualifies.

Reply:
While I don't disagree with the RFD (except for context.TODO() which I think is always a bad choice), I do disagree with this specific change. I would rather not log that particular line than change the signature in a way that gives a wrong impression about what the methods do.

Reply:
Approved - I won't block the refactor but as stated before I would very much prefer not to change the signatures in this file.